src/Store.cs

```diff
@@ -232,6 +240,12 @@ public Store(Engine engine)
         /// </summary>
         /// <param name="config">The WASI configuration to use.</param>
         public void SetWasiConfiguration(WasiConfiguration config) => ((IStore)this).Context.SetWasiConfiguration(config);
+        
+        /// <summary>
+        /// Configures the relative deadline at which point WebAssembly code will trap.
+        /// </summary>
+        /// <param name="ticks_beyond_current"></param>
+        public void SetEpochDeadline(ulong ticks_beyond_current) => ((IStore)this).Context.SetEpochDeadline(ticks_beyond_current);
```

Comment:
Naming convention nit: can we rename `ticks_beyond_current` to `ticksBeyondCurrent` for the [C# coding convention](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#camel-case)?